qutebrowser/misc/utilcmds.py

```diff
@@ -229,18 +229,34 @@ def debug_trace(expr=""):
 
 
 @cmdutils.register(maxsplit=0, debug=True, no_cmd_split=True)
-def debug_pyeval(s, quiet=False):
+def debug_pyeval(s, file=False, quiet=False):
     """Evaluate a python string and display the results as a web page.
 
     Args:
         s: The string to evaluate.
+        file: Interpret s as a path to file.
         quiet: Don't show the output in a new tab.
     """
-    try:
-        r = eval(s)
-        out = repr(r)
-    except Exception:
-        out = traceback.format_exc()
+    if file:
+        quiet = True
+        path = os.path.expanduser(s)
+        message.info(path)
```

Comment:
Did you intend to leave this in? I don't think it makes much sense to show a message with a path here.